src/metabase/driver/athena.clj

```diff
@@ -90,10 +89,13 @@
     :varchar    :type/Text} database-type))
 
 ;;; ------------------------------------------------- date functions -------------------------------------------------
-(defmethod unprepare/unprepare-value [:athena Date] [_ value]
-  (unprepare/unprepare-date-with-iso-8601-fn :from_iso8601_timestamp value))
+(defmethod unprepare/unprepare-value [:presto OffsetDateTime]
```

Comment:
This (and below) shouldn't be `:presto`...verify that we actually need `unprepare-value` here.